Bottom: 3a2e44d8111b970d55784ec19f29e5e3856d29ff
Top:    9a1037e29aa5728053312c00cc6ce1654f82303b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 00:28:55 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 3468b9b..399b614 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
